core/client/runtime/pom.xml

```diff
@@ -162,6 +162,23 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.coderplus.maven.plugins</groupId>
+        <artifactId>copy-rename-maven-plugin</artifactId>
```

Comment:
Would it be possible to use one of our existing plugins?

Reply:
discussed offline. the current resources plugin can not rename, and we want to keep the convention that in `target` dir, uber jars are named after `*-jar-with-dependencies.jar`